docs/README.md

```diff
@@ -2,7 +2,7 @@
 
 ![](./assets/banner-search.png)
 
-The Search module provides a mirrored index of all CMS content that is optimized for search relevance, speed and accuracy. The Search module overrides the default search functionality to query the specialized search index in place of a standard MySQL query. This means all default search operations using the CMS search APIs for posts such as `WP_Query`, `get_posts`, `get_search_form()` and the REST API search endpoints will transparently make use of the search index.
+The Search module provides a mirrored Elasticsearch index of all CMS content that is optimized for search relevance, speed and accuracy.
```

Comment:
Do we want to mention something about it differing from MySQL?

```suggestion
The Search module provides a mirrored Elasticsearch index of all CMS content that is optimized for search relevance, speed and accuracy. This operates at a higher level than the primary datastore (MySQL) and is the recommended solution for searching.
```

or something like that; we mention queries can be done in MySQL below, so seems we should have something here

Reply:
Yeah. I'll add more info as to the benefits of ES as well.